src/client/mod.rs

```diff
@@ -288,6 +288,19 @@ impl Client {
         }
     }
 
+    pub(crate) async fn watch_helper(
+        &self,
+        pipeline: impl IntoIterator<Item = Document>,
+        mut options: Option<ChangeStreamOptions>,
+        target: ChangeStreamTarget,
+    ) -> Result<ChangeStream> {
+        let pipeline: Vec<Document> = pipeline.into_iter().collect();
+        resolve_options!(self, options, [selection_criteria]);
+
+        let watch = Watch::new(target, pipeline.clone(), options.clone())?;
+        let (change_stream_spec, session) = self.execute_cursor_operation(watch).await?;
+        Ok(ChangeStream::new(change_stream_spec, session, self.clone()))
+    }
```

Comment:
add a newline after this brace to give docstring below some space

Reply:
fixed